<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PentIT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabMainLog">
       <attribute name="title">
        <string>Основное</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEditLog">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Здесь отображаются те пакеты программ, которые уже установлены в вашей системе и те, которые необходимо установить дополнительно&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; color:#000000;&quot;&gt;Проверка подключенных пакетов&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; color:#000000;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextBrowserInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="actionAirmon_ng"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Опции</string>
    </property>
    <addaction name="actionOpen_Log"/>
    <addaction name="separator"/>
    <addaction name="actionMAC_Changer"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAbout_this_Program"/>
    <addaction name="actionAbout_Aircrack_ng"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAirmon_ng">
   <property name="text">
    <string>Выбрать интерфейс</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionMAC_Changer">
   <property name="text">
    <string>Изменить MAC-адрес</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/Nokia-QtCreator-64.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>About_QT</string>
   </property>
  </action>
  <action name="actionAbout_this_Program">
   <property name="text">
    <string>О приложении</string>
   </property>
  </action>
  <action name="actionAbout_Aircrack_ng">
   <property name="text">
    <string>О Aircrack-ng</string>
   </property>
  </action>
  <action name="actionGeneral_options">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/Settings.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Основные настройки</string>
   </property>
  </action>
  <action name="actionCheck_Updates">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/Update.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Check Updates</string>
   </property>
  </action>
  <action name="actionOpen_Log">
   <property name="text">
    <string>Открыть статистику</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
